### PR TITLE
#2713 - scroll MyAccount Overlay

### PR DESCRIPTION
**Pull request by tatiana-scandi**
Issue: https://github.com/scandipwa/scandipwa/issues/2713

### CODE REVIEW

File: packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.style.scss [hunk 1]
```diff
@@ -172,6 +172,11 @@
         display: grid;
         grid-template-rows: max-content 1fr;
 
+        @include desktop {
+            overflow-y: scroll;
+            max-height: calc(100vh - var(--header-nav-height) - 40px);
```
Comment (IvansZuks) [inline packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.style.scss:177]: Why 40px? Is it height of some block?

Comment (tatiana-scandi) [reply packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.style.scss:177]: It is distance from Overlay popup bottom to bottom of viewport. It looks nicer when there is spacing between overlay and bottom of the page.

Comment (IvansZuks) [reply packages/scandipwa/src/component/MyAccountOverlay/MyAccountOverlay.style.scss:177]: Would be great to move this 40px to variable with name which will make sense. Or at least leave comment about this value

**Review by zans-laksa — APPROVED** [review]
Code review passed.

**Pull request merged**